osm2gtfs/tests/core/tests_creator_factory.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+
+import unittest
+import os
+from osm2gtfs.core.configuration import Configuration
+
+
+class TestCoreCreatorFactory(unittest.TestCase):
+
+    def test_naming_convention(self):
+
+        path = os.path.dirname(__file__) + "/../../creators/"
+        creators = ['agency', 'feed_info', 'routes', 'schedule', 'stops', 'trips']
+
+        # pylint: disable=unused-variable
+        for subdir, dirs, files in os.walk(path):
+
+            # Loop through available directories of creators
+            for selector in dirs:
+
+                # Check if creator directory fits naming convention
+                self.assertTrue(
+                    self._check_naming_convention(selector),
+                    'The path "creators/' + selector +
+                    ' " doesn\'t fit the naming convention.')
+
+                # Check for existing configuration file
+                config_file_path = path + selector + "/" + selector + ".json"
```

Comment:
Why has it to be `ni_esteli/ni_esteli.json` for the config file? I think it would be a lot more simple to change this naming convention to only `config.json` in all the creators subfolders, so you don't have to type the (potentially long) selector name twice while running _osm2gtfs_.

I see this would be another change added to this PR, but as you're already renaming the json files, I think it would be the right place.

Reply:
Yes, I agree with you. It would be nicer, but I wanted to keep this PR smallest possible. However as you are also suggesting it, I will apply it to the next rebase here.

Reply:
Thanks a lot!